docs/pipelines-in-pipelines.md

```diff
@@ -0,0 +1,106 @@
+<!--
+---
+linkTitle: "Pipelines in Pipelines"
+weight: 406
+---
+-->
+
+# Pipelines in Pipelines
+> 🌱 Pipelines in Pipelines is an alpha feature. The enable-api-fields feature flag must be set to "alpha" to specify PipelineRef or PipelineSpec in a PipelineTask. This feature is in Preview Only mode and not yet supported/implemented
+
+* [Proposal](https://github.com/tektoncd/community/blob/main/teps/0056-pipelines-in-pipelines.md "Proposal")
+* [Summary](#summary)
+* [Specification](#specification)
+* [Parameters](#parameters)
+
+## Summary
+A mechanism to define and execute Pipelines in Pipelines, alongside Tasks and Custom Tasks
+
+## Specification
```

Comment:
again, rename this to Specifying `pipelineRef` in `Tasks`

Reply:
Here is a reference: https://github.com/tektoncd/pipeline/blob/main/docs/matrix.md#specifying-results-in-a-matrix

Tekton Pipelines doc follows [these](https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md) conventions to make the doc consistent throughout the project. Please follow the same convention. Change

```
### Specification
```

 to:

```
### Specifying `pipelineRef` in `Tasks`
```

Reply:
Sounds good